units/src/iosMain/kotlin/dev/icerock/moko/units/UnitCollectionViewDataSource.kt

```diff
@@ -7,24 +7,25 @@ package dev.icerock.moko.units
 import kotlinx.cinterop.ExportObjCClass
 import platform.Foundation.NSIndexPath
 import platform.UIKit.UICollectionView
-import platform.UIKit.UICollectionViewCell
 import platform.UIKit.UICollectionViewDataSourceProtocol
 import platform.UIKit.row
 import platform.darwin.NSInteger
 import platform.darwin.NSObject
+import platform.UIKit.UICollectionViewCell as UICollectionViewCell
 
 @ExportObjCClass
-class UnitCollectionViewDataSource(
-    private val collectionView: UICollectionView
-) : NSObject(), UICollectionViewDataSourceProtocol {
+class UnitCollectionViewDataSource internal constructor(collectionView: UICollectionView): NSObject(), UICollectionViewDataSourceProtocol {
     private val unitsRegistry = UnitsRegistry<UICollectionView, CollectionUnitItem>(collectionView)
     var unitItems: List<CollectionUnitItem>? = null
         set(value) {
             field = value
             if (value != null) unitsRegistry.registerIfNeeded(value)
-            collectionView.reloadData()
```

Comment:
i think it's should be automatic. later we just replace this `reloadData` to animated items update with diff by `itemId`